app/javascript/packs/ajax.js

```diff
@@ -67,61 +67,75 @@ global.ajaxCall = (
     '<div class="loading"><i class="fas fa-compass"></i></div>'
   );
   // Make a remote call to the target url (e.g. search, show, etc.)
-  $.ajax({
-    type: "GET",
-    url: target,
-    data: valuesToSubmit,
-    dataType: "html",
-    success: function (text) {
-      // Parse received text to valid DOM elements and insert them into the #browse-outlet
-      const parsed = new DOMParser().parseFromString(text, "text/html");
-      const content = parsed.querySelector("#app-outlet-outer");
-      $("#browse-outlet").html(content.innerHTML);
-
-      // Retrieve all location info from a (possibly nonexistent) script tag on the
-      // target page and show it on the map if it exists
-      const latLongInfo = content.querySelector("#_latlonginfo");
-      setTimeout(() => {
-        if (latLongInfo) {
-          const latlong = JSON.parse(latLongInfo.innerHTML);
-          showTargetMarker(latlong);
+    $.ajax({
```

Comment:
What exactly are the changes in the following lines?
Is this an indentation issue? If thats the case could you revert the indentation changes?
If any other branches have changes in ajax.js it becomes much harder to see whether conflicts are actual conflicts or just indentation differences.